### PR TITLE
KK-45 | Add preliminary form to validate supported user.

### PR DESCRIPTION
**Pull request by midori0507**
- Add form using formik.
- Create reusable `Field` elements.
- Add basic validation and error handler.

### CODE REVIEW

File: src/common/components/form/validationUtils.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+} from '../../time/constants';
+
+/** validateBirthDay()
+ * Validate user input child birthday.
+ * This app only target recently born child, so we support child which born after 2019 only.
+ * Input time which come from future is not accepted
+ * @param value Input value.
+ */
+const validateBirthDay = (value: string | number | undefined) => {
+  if (!value) {
```
Comment (badrange) [inline src/common/components/form/validationUtils.tsx]: How can we test this service before the year 2020?

File: src/common/components/form/validationUtils.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  const nowMoment = newMoment();
+
+  if (!inputMoment.isValid()) {
+    return formatMessage('validation.date.invalidFormat');
+  }
+
+  const supportedStart = newMoment(SUPPORTED_START_BIRTHDAY);
+
+  if (inputMoment < supportedStart || inputMoment > nowMoment) {
+    return formatMessage('validation.date.nonSupported');
```
Comment (badrange) [inline src/common/components/form/validationUtils.tsx]: `notSupported` sounds more proper English like to my ear

File: src/domain/app/constants.tsx [hunk 3]
```diff
@@ -0,0 +1,3 @@
+export enum SUPPORTED_CITY {
+  HELSINKI = 'Helsinki',
```
Comment (badrange) [inline src/domain/app/constants.tsx]: Remember that the :trollface: Swedish speakers say *Helsingfors*

File: src/common/components/form/validationUtils.tsx (hunk 2 above)
Comment (badrange) [reply src/common/components/form/validationUtils.tsx]: The word exists in the dictionary though:
https://en.wiktionary.org/wiki/nonsupported

and unsupported might be even better!

File: src/common/components/form/validationUtils.tsx (hunk 1 above)
Comment (midori0507) [reply src/common/components/form/validationUtils.tsx]: i thought we only support kids born from beginning of 2019 -> now, not in future or 2018?

File: src/common/components/form/validationUtils.tsx (hunk 2 above)
Comment (midori0507) [reply src/common/components/form/validationUtils.tsx]: Will do!

**Pull request merged**